Formula/jmeter.rb

```diff
@@ -22,7 +24,7 @@ def install
     rm_f Dir["bin/*.bat"]
     prefix.install_metafiles
     libexec.install Dir["*"]
-    (bin/"jmeter").write_env_script libexec/"bin/jmeter", JAVA_HOME: Formula["openjdk"].opt_prefix
+    (bin/"jmeter").write_env_script libexec/"bin/jmeter", JAVA_HOME: Formula["openjdk@11"].opt_prefix
```

Comment:
yeah, maybe also allow the `JAVA_HOME` to be overridden.

```suggestion
    (bin/"jmeter").write_env_script libexec/"bin/jmeter", Language::Java.overridable_java_home_env("11")
```

Reply:
most definitely not

Reply:
According to https://community.snowflake.com/s/article/JDBC-Driver-Compatibility-Issue-With-JDK-16-and-Later adding --add-opens java.base/java.nio=ALL-UNNAMED and -Djdk.module.illegalAccess=permit (or --illegal-access=permit) may help. There's a section in jmeter.sh where you could add these arguments. Have they tried it?

Reply:
I have provided this information too. But only information I have from them is that the downgrade fix solved the problem. We're in a different time zones which makes communication a bit difficult. I was opening this PR myself because they seemed uninterested in doing so once they were unblocked so I guess it's likely that they tried jdk 11 first which solve the problem and that was enough for them.

This solution is also one of the proposed in upstream issue I linked above https://github.com/snowflakedb/snowflake-jdbc/issues/589#issuecomment-983944767 so it's likely it would also do the trick.

My understanding is that upstream library is yet to implement support for JDK 17 so downgrade seemed as a safer choice to me anyway. Obviously the most sound option would be for upstream to patch the issue and release new version.

Maybe someone else can test that? In any case I hope for the fix to the issue upstream so hopefully any fix in homebrew would be just temporary.